docs/current_docs/features/visualization.mdx

```diff
@@ -0,0 +1,75 @@
+
+The TUI renders a tree of Dagger API calls, represented as GraphQL queries. A parent-child relationship means that the child call is being made by the parent call.
+- A red X or green check indicates whether the call succeeded or failed.
+- A call's duration is rendered after its name.
+
+The CLI has tiered verbosity: `-v`, `-vv`, `-vvv` (just like `curl`):
+- `-v` keeps spans visible after they complete, rather than disappearing after a brief pause;
+- `-vv` reveals internal and encapsulated spans;
+- `-vvv` reveals sub-100ms spans.
+
+For additional debugging information, add the `--debug` flag to the `dagger call` command.
+
+## Traces
+
+Dagger Cloud provides Traces, a browser-based interface focused on tracing and debugging Dagger pipeline runs. A Trace represents one invocation of a Dagger Function, run locally or in CI. It contains detailed information about the operations performed by the Dagger Function.
+
+Traces provides a UI to debug what went wrong when something fails, focusing on tracing what happened and when it happened. Traces also helps you optimize your Dagger Functions as you author them, by giving you the detailed step-by-step execution of your pipeline and the associated telemetry. This is equally useful for debugging pipelines running on a developer's local environment and for pipelines in CI environments.
+
+Here's an example of it in action:
+
+![Dagger Trace](/img/current_docs/quickstart/trace.gif)
+
+Dagger is aware of the context it runs on. The Dagger Cloud dashboard classifies Traces depending on whether you triggered a run from your local development host or from your CI environment. You can select the appropriate view from the dashboard sidebar. Here's an example showing Traces generated in a CI environment:
+
+![Traces list](/img/current_docs/features/trace-list.png)
+
+This initial view displays groups of Traces that belong to the same point in Git history. The grouping is determined by the branch/reference combination where Dagger was triggered. This is useful when you need to inspect all the Traces emitted at a specific, pinned point in the repository's version history
+
+Here's a quick summary of what you'll see for each Trace group:
+
+- Repository name
+- Commit metadata, including comment and commit author
+- Indication of the run status: in progress, succeeded, or failed
+- The command used
+- The run start time
+- The run duration
+
+:::important
+All of the information listed above may not be available for a specific group; it differs depending on whether the Trace is generated by a local run or a CI run. When Dagger runs in a CI context like GitHub, GitLab, CircleCI, or Jenkins, additional Trace metadata is displayed based on the Git repository information available.
```

Comment:
I wonder if here we want to add a warning admonition as well about orphaned traces.
For groups to be properly set, the branch and commit SHA have to be available to the engine. If the repository is not correctly provisioned to the engine, it's likely that wrong or partial Git metadata will be displayed. When this happens (traces ran in a CI context, but with incomplete Git metadata), they'll be marked as orphaned, and **will not be part of a group**.

Reply:
Added